std_msgs/Header header 
string[] nodes
duration[] jitter
duration[] latency
